implementations/rust/ockam/ockam_command/src/command_global_opts.rs

```diff
@@ -47,9 +47,10 @@ impl CommandGlobalOpts {
         global_args: &GlobalArgs,
         cmd: &OckamSubcommand,
     ) -> miette::Result<Self> {
-        let terminal = Terminal::from(global_args);
+        let mut terminal = Terminal::from(global_args);
         let logging_configuration =
             Self::make_logging_configuration(global_args, cmd, terminal.clone().stdout().is_tty())?;
+        terminal.set_logging_enabled(logging_configuration.is_enabled());
```

Comment:
Could we swap the lines 50 and 51 and pass the `logging_configuration.is_enabled()` boolean directly to create a `Terminal` instead of having a mutable method?

Reply:
The problem is that the `make_logging_configuration` needs to check the terminal... I could recreate the instance as well to avoid the mutation.